ui/loading_indicators.py

```diff
@@ -42,6 +42,10 @@ def get_centered_text_bounds(draw, content, device_size):
         dw, dh = device_size
         return Rect(dw / 2 - w / 2, dh / 2 - h / 2, dw / 2 + w / 2, dh / 2 + h / 2)
 
+    @staticmethod
```

Comment:
This seems to be an unrelated change that got into this PR =) I'll merge it, it's no harm, but it looks like it belongs to something else.

Reply:
Oh wait, now I see - sorry!